mkosi/__init__.py

```diff
@@ -3725,20 +3764,61 @@ def make_tar(args: CommandLineArguments, root: str, do_run_build_script: bool, f
     if for_cache:
         return None
 
-    tar_root_dir = f"{root}/usr" if args.usr_only else root
+    root_dir = f"{root}/usr" if args.usr_only else root
 
     with complete_step("Creating archive"):
         f: BinaryIO = cast(BinaryIO, tempfile.NamedTemporaryFile(dir=os.path.dirname(args.output), prefix=".mkosi-"))
         # OpenMandriva defaults to bsdtar(libarchive) which uses POSIX argument list so let's keep a separate list
         if shutil.which("bsdtar") and args.distribution == Distribution.openmandriva:
-            _tar_cmd = ["bsdtar", "-C", tar_root_dir, "-c", "-J", "--xattrs", "-f", "-", "."]
```

Comment:
(btw, i think to keep things systematic and well-defined on all distros we should always use "gtar" here first, and fall back to "tar", but never use bsdtar. In particular as bsdtar has no selinux control iirc)

but that's something to fix separately.